test/utils/utils.go

```diff
@@ -163,6 +159,14 @@ func Kubectl(args ...string) {
 	}
 }
 
+func KubectlDelete(args ...string) {
+	deleteArgs := []string{
+		"delete", "--wait=false", "--ignore-not-found",
```

Comment:
I was hoping `--wait=false` would save some time, but it looks like the time to run the tests is similar.